shared/model/response.ts

```diff
@@ -8,7 +8,7 @@ export default class ResponseEntity<T> {
     errors?: Array<string>;
     data?: T;
 
-    constructor(params: ReponseEntityParams<T> = {}) {
+    constructor(params: ResponseEntityParams<T>) {
```

Comment:
Default parameter was implemented to make it possible to create blank responses in case method shoudn't return value, but may return errors (or not)

Reply:
return null